src/javatests/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/ExtractSubcommandTest.java

```diff
@@ -150,6 +150,36 @@ public void call_successWithSqlScripts() throws IOException, SQLException {
     assertThat(arguments.schemaFilters()).isEmpty();
   }
 
+  @Test
+  public void call_successWithNoNeedQueryText() throws IOException, SQLException {
+    ExtractExecutor executor = Mockito.mock(ExtractExecutor.class);
+    CommandLine cmd = new CommandLine(new ExtractSubcommand(() -> executor, scriptManager));
+
+    assertThat(
+            cmd.execute(
+                "--db-address",
+                "jdbc:hsqldb:mem:my-db1.example",
```

Comment:
there are 20+ tests, each having `--db-address`/`--db-user`/`--dp-password`. For most of the tests, such a setup is not really relevant to reader and it's good idea to hide it. Consider extracting it to a helper method

Also, we agreed to separate act and assert steps

Reply:
I'd say in this particular case the benefit of making this part DRY (minus the effort) does not beat the benefit of keeping this DAMP (go/tott/598).

Changed both this and other tests' arrangement to conform with our test structure convention.

Reply:
DAMP principle should be applied to parts, being accessed during the test. Required parameters are not accessed (except tests, explicitly checking the application fails if no required parameters are passed)

I would agree we need redundancy if we would need to check the value of passed db address in every test. In that case, hiding it in the helper method may be misleading

Reply:
I see your point regarding the values being accessed, though I think there's another aspect to it: the effort required to keep the uniqueness. It's not as clear cut in this case as I'm not sure these 6 lines repeated through out tests are much less readable than a separate method that populates those three fields  (compared to, for example, https://github.com/google/dwh-assessment-extraction-tool/pull/94/files/b2efaee661e78c4db332959cccb0f6c3acd464f8#r826105482 where it's pretty clear that the repeated definitions create too much redundancy for readers' eyes.)